app/views/common/_event_social_buttons.html.erb

```diff
@@ -19,10 +19,13 @@
 	</div>
 
 	<div>
-		<a class='button--circle--large grey'>
-			<!--= on 'click' (open_modal_if_confirmed 'shareEmail') -->
+		<%= mail_to('',
+			subject: "Check the #{@event.name} event from #{@nonprofit.name}!",
+			body: "Hi,\n\n#{@event.summary.present? ? [@event.summary, "\n\n"].join('') : ''}Follow this link for the event and participate: #{url}",
```

Comment:
Does this work the same? I think it's clearer

```suggestion
			body: "Hi,\n\n#{@event.summary.present? ? "#{@event.summary}\n\n" : ''}Follow this link for the event and participate: #{url}",
```

Reply:
It does! Nice catch, thanks